qutebrowser/browser/commands.py

```diff
@@ -870,37 +870,44 @@ def _yank_to_target(self, s, sel, what, keep):
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
     @cmdutils.argument('count', count=True)
-    def zoom_in(self, count=1):
+    @cmdutils.argument('quiet', flag='q')
+    def zoom_in(self, count=1, quiet=False):
         """Increase the zoom level for the current tab.
 
         Args:
             count: How many steps to zoom in.
+            quiet: Don't show information message with result.
```

Comment:
This reads a bit weird - what about "Don't show a zoom level message."?

(same for all three changes)